webapp/home/subdomains/proteomics/content/database_searching.py

```diff
@@ -44,7 +44,7 @@
         "inputs": [
             {
                 'datatypes': ['thermo.raw, mzML, mzXML'],
-                'label': 'Thermo raw, mzML, or mzXML files'
+                'label': 'Mass spectrometry data sets: Thermo raw, mzML, or mzXML files'
```

Comment:
Sorry, can you delete `: Thermo raw, mzML, or mzXML files` as this is just a repeat of the `datatype` information